client/src/components/navbar.css

```diff
@@ -58,6 +68,10 @@
     align-items: center;
 }
 
+.active {
```

Comment:
Active is a very broad className.  Should be more descriptive.   It looks to me that its toggling visibility rather making something active. Maybe 'is-visible'?